cordova-js-src/splashscreen.js

```diff
@@ -148,29 +152,27 @@ function init(config, manifest) {
     extendedSplashProgress.classList.add('win-medium');
     extendedSplashProgress.classList.add('win-ring');
 
+    extendedSplashImage.style.maxWidth = "100%";
+    extendedSplashImage.style.maxHeight = "100%";
+    extendedSplashImage.src = splashImageSrc;
+
     if (isWp81 || isWp10) {
```

Comment:
nit: Maybe change this condition to `if (isPhoneDevice) {...` where `isPhoneDevice = isWp81 || isWp10` - just to improve readability